config/environments/production.rb

```diff
@@ -110,13 +110,13 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 
-  config.app_domain = ENV["APP_DOMAIN"] || "localhost:3000"
+  # config.app_domain = ENV["APP_DOMAIN"] || "localhost:3000"
```

Comment:
hm, is this a temporary comment or unintentional and meant to be removed? if it's intentional, i'd add a TODO here to explain what it is there for/when it can be removed

Reply:
Deleted this code, thanks.